genotype_variants/create_duplex_simplex_dataframe.py

```diff
@@ -67,9 +67,12 @@ def create_duplex_simplex_dataframe(simplex_dataframe, duplex_dataframe):
         exit(1)
 
     try:
```

Comment:
This try and except statements are for merging purpose, can you have them after the if shape statement

```
if shape>0:
   try:
        merge
   except:

else:
```